### ISSUE TITLE
Move nag scripts in subdirectories

### ISSUE THREAD

**Issue by marco-c** [mozilla/relman-auto-nag#572]
This will make it easier to find all scripts related to a given topic.
E.g. all scripts related to the bug type could be in a bug_type directory.

**Comment by calixteman** [comment]
yeah, we could have directories related to the action (e.g. change bug_type, needinfo foobar, ...).
